README.rst

```diff
@@ -14,19 +14,23 @@ Project DeepSpeech
 
 DeepSpeech is an open source Speech-To-Text engine, using a model trained by machine learning techniques based on `Baidu's Deep Speech research paper <https://arxiv.org/abs/1412.5567>`_. Project DeepSpeech uses Google's `TensorFlow <https://www.tensorflow.org/>`_ to make the implementation easier.
 
-**NOTE:** This documentation applies to the **MASTER version** of DeepSpeech only. **Documentation for the latest stable version** is published on `deepspeech.readthedocs.io <http://deepspeech.readthedocs.io/?badge=latest>`_.
+To install and use the **latest stable version**, please see the `DeepSpeech install guide <http://deepspeech.readthedocs.io/?badge=latest>`_  available at `deepspeech.readthedocs.io <http://deepspeech.readthedocs.io/?badge=latest>`_.
 
-To install and use deepspeech all you have to do is:
+To install the current release:
```

Comment:
Maybe wording it as "current **stable**" release then ?

Reply:
That makes sense -- updated